userguide/dailyuse/libertine.rst

```diff
@@ -105,78 +115,96 @@ Tips
 
-For example run::
+For example, run::
 
   libertine-launch -i CONTAINER-IDENTIFIER ls -a
 
-To get a shell as user ``phablet`` run::
+To get a shell as the user ``phablet``, run::
 
   DISPLAY= libertine-launch -i CONTAINER-IDENTIFIER /bin/bash
 
 .. note::
-    When you launch bash in this way you will not get any specific feedback to confirm that you are now *inside* the container. You can check ``ls /`` to confirm for yourself that you are inside the container. The listing of ``ls /`` will be different inside and outside of the container.
+    When you launch Bash in this way you will not get any specific feedback to confirm being *inside* the container.
+    You can check ``ls /`` to confirm for yourself you are actually inside the container.
+    The listing of ``ls /`` will be different inside and outside the container.
 
 Accessing SD card
 ^^^^^^^^^^^^^^^^^
 
-In order to access your SD-card or any other part of the regular filesystem from inside your libertine container you must create a bind mount.
+To access your SD-card or any other part of the regular filesystem from inside your Libertine container you must create a bind mount.
 
-In order to add a bind mount use::
+To add a bind mount, use::
 
   libertine-container-manager configure -i CONTAINER-IDENTIFIER -b add -p /media/phablet/ID-OF-SD
   
-You can also make deep links in case you only want parts of your SD-card available in the container. In this case just the entire path to the directory you want to bind mount::
+You can also make deep links if you only want parts of your SD-card available in the container.
+In this case just add the entire path to the directory you want to bind mount::
 
   libertine-container-manager configure -i CONTAINER-IDENTIFIER -b add -p /media/phablet/ID-OF-SD/directory/you/want
   
-This will not allow the container access to any of the directories earlier in the path for anything other than accessing your mounted directory.
+This will not give the container access to any of the directories earlier in the path for anything other than accessing your mounted directory.
     
-In order to use the SD-card as extra space for your container, make sure first to format it using ext4 or similar.
-There is a mis-feature in udisk2 that mounts SD-cards (showexec) that ensures only files ending in .bat, .exe or .com can be executed from the drive if it is (v)fat formatted. This has been changed in other distributions allowing any file to have execute priviliges, but not ubuntu. The reccomended workaround is to add a udev rule to control how to mount a card with a given id, but since the udev rules are on the read only port on touch, this is not possible.
+In order to use the SD card as extra space for your container, make sure to first format it using ext4 or similar.
+There is a mis-feature in UDisks2 that mounts SD-cards (showexec), ensuring only files ending in .bat, .exe or .com can be executed from the drive if it is (V)FAT formatted.
```

Comment:
Not 100% sure about UDisks2

Reply:
lower case and with plural s https://packages.ubuntu.com/xenial/udisks2 https://www.freedesktop.org/wiki/Software/udisks/

Reply:
But then storaged.org/doc/udisks2-api/latest/ and https://github.com/storaged-project/udisks

Reply:
True! 

Flip a coin :D

Reply:
I prefer the latter since the actual project uses it. However consistently, I don't know.